### PR TITLE
feat: Expose `downloadBinary` function to install binary

### PR DESCRIPTION
**Pull request by lforst**
We would like to be able to install the CLI manually in downstream dependencies in case postinstall scripts did not get executed.

### CODE REVIEW

**Review by Lms24 — APPROVED** [review]
Nice! I believe this was the actual download logic is more or less C&P from what we had before, correct? 

(fwiw, TS would probably make sense but let's not overcomplicate things)

**Comment by lforst** [comment]
> I believe this was the actual download logic is more or less C&P from what we had before, correct?

Correct. I didn't change anything except for adding the logger thing.

File: js/install.js [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+      throw new Error(`Unexpected sentry-cli version "${version}", expected "${expected}"`);
+    }
+  });
+}
+
+function downloadBinary(logger = ttyLogger) {
+  if (process.env.SENTRYCLI_SKIP_DOWNLOAD === '1') {
+    logger.log(`Skipping download because SENTRYCLI_SKIP_DOWNLOAD=1 detected.`);
+    return;
+  }
```
Comment (Jonir227) [inline js/install.js:199]: this should return Promise. it makes error on flag 1

Comment (lforst) [reply js/install.js:199]: Ah right. Good catch!

**Pull request merged**